src/markerclusterer.ts

```diff
@@ -180,8 +180,40 @@ export class MarkerClusterer extends OverlayViewSafe {
 
       // allow algorithms to return flag on whether the clusters/markers have changed
       if (changed || changed == undefined) {
+        const records = new Map<{ lat: number; lng: number }, Cluster>();
```

Comment:
![image](https://github.com/googlemaps/js-markerclusterer/assets/248818/456c571c-c9a3-4f00-b9b2-5f937e6efd64)

That does not work in JS.

Reply:
[b0bf346](https://github.com/googlemaps/js-markerclusterer/pull/536/commits/b0bf346064a9a209cc3135d7eb8b36b369ad7574)